cabal.project

```diff
@@ -1,3 +1,15 @@
 -- HI! for pants reasons, if you add anything load bearing in here, you have to
--- add it to raw-project in cabal.haskell-ci
+-- add it to `generate cabal.project` in `.github/workflows/haskell-ci.yml`
+source-repository-package
+  type: git
+  location: https://github.com/nikita-volkov/refined
+  tag: 11296288b5e2f2b391ee721b50af3cfe7beaa790
```

Comment:
I think we should be able to get a release for whatever we need here - especially if it's just a version bump or similar. 

https://github.com/nikita-volkov/refined/commit/11296288b5e2f2b391ee721b50af3cfe7beaa790 This is the comment in question. It bumps the `aeson` version bound. We could instead specify an `allow-newer` here and request release + request a Hackage revision if one could be made.

Reply:
OK - I think we'll be fine to release without this, and instead say `allow-newer` when we consume the library and want the aeson.

https://github.com/nikita-volkov/refined/issues/102

I made an issue for a release/revision that should make it seamless.

For now, we can test with an `allow-newer`:

https://cabal.readthedocs.io/en/3.4/cabal-project.html#cfg-flag---allow-newer

```
allow-newer:
  refined:aeson
```